packages/cart-sdk/hooks/use-update-line-item-quantity.ts

```diff
@@ -91,6 +93,9 @@ export function useUpdateLineItemQuantity() {
                cartKeys.cart,
                context?.previousCart
             );
+            addErrorMessage(
+               (error as string) || 'An error occurred updating the cart.'
+            );
```

Comment:
What do you think about passing an error object instead of a string? This would keep the solution open for extensions (by extending error one could pass more information if needed) and would align with the approach used for other error handling situation (e.g. [error boundaries](https://nextjs.org/docs/app/api-reference/file-conventions/error))

```suggestion
            if (error instanceof Error) {
               addErrorMessage(error);
            } else {
               addErrorMessage(new Error('An error occurred updating the cart.'));
            }
```